terraform/cloud-platform-components/nginx-ingress-acme.tf

```diff
@@ -180,3 +182,37 @@ resource "null_resource" "nginx_ingress_servicemonitor" {
     contents = "${sha1(file("${path.module}/resources/nginx-ingress/servicemonitor.yaml"))}"
   }
 }
+
+resource "null_resource" "nginx_ingress_errors_deployment" {
+  depends_on = ["helm_release.prometheus_operator"]
```

Comment:
I might be missing something but why does it depend on prometheus?

Reply:
Thanks @jasonBirchall , fixed it to depends on  to "helm_release.nginx_ingress_acme"